### PR TITLE
doc update

### PR DESCRIPTION
**Pull request by slimslenderslacks**

### CODE REVIEW

File: INFO.md [hunk 1]
```diff
@@ -31,17 +31,25 @@ the repository's default branch on your behalf. Note that this requests the pull
 request creation in draft mode, essentially marking it as not yet ready to be
 reviewed.
 
+In the example above, we did not put the `/pr` command on the first line of the
+commit message. This means that it will not show up in the "short" format of the
+commit log. Users may chooto remove these command lines during squash, or
```
Comment (cdupuis) [inline INFO.md:36]: did you mean to write _choose to_ instead of _chooto_?

**Review by cdupuis — COMMENTED** [review]
Typo, perhaps?

File: INFO.md (hunk 1 above)
Comment (slimslenderslacks) [reply INFO.md:36]: yes, choose is better than chooto

File: INFO.md [hunk 2]
```diff
@@ -31,17 +31,25 @@ the repository's default branch on your behalf. Note that this requests the pull
 request creation in draft mode, essentially marking it as not yet ready to be
 reviewed.
 
+In the example above, we did not put the `/pr` command on the first line of the
+commit message. This means that it will not show up in the "short" format of the
+commit log. Users may choose remove these command lines during squash, or
```
Comment (cdupuis) [inline INFO.md:36]: hmm, isn't there still a tiny issue:

```suggestion
commit log. Users may choose to remove these command lines during squash, or
```

**Review by cdupuis — APPROVED** [review]

**Pull request merged**